--- wordpress-mu-2.8.6/index-install.php~	2009-11-19 14:39:53.000000000 +0200
+++ wordpress-mu-2.8.6/index-install.php	2009-11-19 14:42:58.870091243 +0200
@@ -288,7 +288,7 @@
 		<h2>Warning!</h2>
 		<div style='border: 1px solid #ccc'>
 			<p style='font-weight: bold; padding-left: 10px'>One or more of the directories must be made writeable by the webserver. You will be reminded to reset the permissions at the end of the install.<br />
-				Please <code>chmod 777 <q>directory-name</q></code> or <code>chown</code> that directory to the user the web server runs as (usually nobody, apache, or www-data)<br />
+				Please <code>chmod 777 <q>directory-name</q></code> or <code>chown</code> that directory to the user the web server runs as (usually "nobody", "apache", or "www-data"; It is "http" in PLD Linux)<br />
 				Refresh this page when you're done!<br /></p>
 		</div>
 		<p>Quick fix:<br /> <code>chmod&nbsp;777&nbsp;<?php
--- wordpress-mu-2.8.6/index-install.php~	2009-11-19 16:54:08.000000000 +0200
+++ wordpress-mu-2.8.6/index-install.php	2009-11-19 16:55:38.075995166 +0200
@@ -359,7 +359,7 @@
 	return true;
 }
 
-function printstep1form( $dbname = 'wordpress', $uname = 'username', $pwd = 'password', $dbhost = 'localhost', $vhost = 'yes', $prefix = 'wp_' ) {
+function printstep1form( $dbname = 'wordpress', $uname = 'mysql', $pwd = '', $dbhost = 'localhost', $vhost = 'yes', $prefix = 'wp_' ) {
 	$weblog_title = ucfirst( $_SERVER[ 'HTTP_HOST' ] ) . ' Blogs';
 	$email = '';
 	$hostname = $_SERVER[ 'HTTP_HOST' ];
